packages/shared/src/components/Markdown.tsx

```diff
@@ -1,12 +1,81 @@
-import React, { ReactElement } from 'react';
+import React, { ReactElement, useEffect, useState } from 'react';
+import ReactDOM from 'react-dom';
+import { tippy } from '@tippyjs/react';
 import { sanitize } from 'dompurify';
 import styles from './markdown.module.css';
+import { ProfileTooltipContent } from './profile/ProfileTooltipContent';
+import { profileTooltipClasses } from './profile/ProfileTooltip';
+import { useProfileTooltip } from '../hooks/useProfileTooltip';
 
-export default function Markdown({
-  content,
-}: {
+const classes = Object.values(profileTooltipClasses).join(' ');
+
+interface MarkdownProps {
   content: string;
-}): ReactElement {
+}
+
+type TippyInstance = ReturnType<typeof tippy>[0];
+
+const getUserPermalink = (username: string) =>
+  `${process.env.NEXT_PUBLIC_WEBAPP_URL}${username}`;
```

Comment:
Just thinking ahead, perhaps we can move this to a more generic file and export it?
This seems like something we'll need more often.

Reply:
I agree this should be in a different place.